javascript-client/package.json

```diff
@@ -26,19 +26,30 @@
     "@typescript-eslint/eslint-plugin": "^4.14.0",
     "@typescript-eslint/parser": "^4.14.0",
     "eslint": "^7.18.0",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-plugin-import": "^2.22.1",
     "npm-run-all": "^4.1.5",
     "rimraf": "^3.0.2",
     "typescript": "^4.1.3"
   },
   "eslintConfig": {
     "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    },
     "plugins": [
       "@typescript-eslint"
     ],
     "extends": [
       "eslint:recommended",
       "plugin:@typescript-eslint/eslint-recommended",
-      "plugin:@typescript-eslint/recommended"
-    ]
+      "plugin:@typescript-eslint/recommended",
+      "airbnb-typescript/base"
+    ],
+    "rules": {
+      "@typescript-eslint/no-non-null-assertion": 0,
```

Comment:
```suggestion
```

`@typescript-eslint/no-non-null-assertion` doesn't currently have any violations in the codebase.

I think it is a good practice to encourage developers to avoid `!.` whenever possible, per the [rationale of the rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-non-null-assertion.md). In some cases, it may indeed be unavoidable, but we can ignore those on a line-by-line basis.

Reply:
Sure thing, I didn't add this, but I'll remove it.